viewing_party/party.py

```diff
@@ -1,23 +1,139 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:

```

Comment:
It can help make code more readable if instead of checking for the conditions we need to run our main logic (here building the dictionary), we instead check for the conditions that would _prevent_ the main logic (if any value is falsy).

Doing so can help us avoid indenting the main logic of our function, which gives it more emphasis that it's the main expected case. Here, this could look like

```py
  if not title or not genre or not rating:
      return None

  # unindented code to build and return movie dictionary
```